syllabus/1-Cryptography/zk-proofs/slides.md

```diff
@@ -0,0 +1,446 @@
+---
+title: ZK proofs
+description: Introduction to zero-knowledge proofs and zk-SNARKS
+duration: 3.5 hour
+---
+
+# ZK Proofs
+
+---
+
+# Outline
+
+<pba-flex center>
+
+1. [ZK Proofs overview](#zk-proofs)
+2. [Examples](#simple-zk-example)
+3. [zk-SNARKs](#zk-snarks)
+4. [Applications of zk-SNARKs](#zk-application)
+5. [Under the hood of zk-SNARKs](#zk-practice)
+6. [Generating ZK-proofs using Circom and snarkjs](#circom-snarkjs)
+
+</pba-flex>
+
+---
+
+## ZK Proofs
+
+How do we do private operations on a public blockchain<br />and have everyone know that they were done correctly?
+
+Notes:
+
+(we are working on substrate support for these and will use them for protocols)
+
+---
+
+## What is a ZK Proof?
+
+- A prover wants to convince a verifier that something is true without revealing why it is true.
+
+- They are mostly interactive protocols, but mostly we'll turn them  non-interactive and deal with their non-interactive variety. 
```

Comment:
Students don't know the definitions of interactive/non-interactive at this point, so probably add a speaker note to explain the distinction briefly before you get to the proper definition later on.